# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1458315358 25200
#      Fri Mar 18 08:35:58 2016 -0700
# Node ID deeb9b4b2846ee3efaf01c1988b3b5eff6e4b9de
# Parent  e9978387898a548858c3c722c5b58cd7ffc94e86
* textscan.h: Add required #includes for types used in header

diff --git a/libinterp/corefcn/textscan.h b/libinterp/corefcn/textscan.h
--- a/libinterp/corefcn/textscan.h
+++ b/libinterp/corefcn/textscan.h
@@ -21,16 +21,25 @@ along with Octave; see the file COPYING.
 */
 
 // @file
 // Implementation of textscan, a versatile text parser.
 
 #if ! defined (octave_textscan_h)
 #define octave_textscan_h 1
 
+#include "octave-config.h"
+
+#include <iosfwd>
+#include <list>
+#include <string>
+
+#include "Cell.h"
+#include "ov.h"
+
 // Delimited stream, optimised to read strings of characters separated
 // by single-character delimiters.
 //
 // The reason behind this class is that octstream doesn't provide
 // seek/tell, but the opportunity has been taken to optimise for the
 // textscan workload.
 //
 // The function reads chunks into a 4kiB buffer, and marks where the
